### PR TITLE
Change of copyright year from 2017 to 2018.

### PR DESCRIPTION
**Pull request by bhargavmehta**
### Description

Have changed files for copyright.
#MM18IN
### Manual testing scenarios

1. 
home page

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/magento2?pullRequest=13027) <br/>All committers have signed the CLA.

File: app/code/Magento/Checkout/view/frontend/web/js/model/shipping-save-processor/payload-extender.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright © 2013-2017 Magento, Inc. All rights reserved.
```
Comment (ishakhsuvarov) [inline app/code/Magento/Checkout/view/frontend/web/js/model/shipping-save-processor/payload-extender.js:2]: Looks like this is an artifact form the previous copyright update. 
Please change it to the one without specific year like in all other code files.

File: app/code/Magento/Customer/view/frontend/web/js/invalidation-processor.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright © 2013-2017 Magento, Inc. All rights reserved.
+ * Copyright © 2013-2018 Magento, Inc. All rights reserved.
```
Comment (ishakhsuvarov) [inline app/code/Magento/Customer/view/frontend/web/js/invalidation-processor.js]: Looks like this is an artifact form the previous copyright update. 
Please change it to the one without specific year like in all other code files.

File: app/code/Magento/Customer/view/frontend/web/js/invalidation-rules/website-rule.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright © 2013-2017 Magento, Inc. All rights reserved.
+ * Copyright © 2013-2018 Magento, Inc. All rights reserved.
```
Comment (ishakhsuvarov) [inline app/code/Magento/Customer/view/frontend/web/js/invalidation-rules/website-rule.js]: Looks like this is an artifact form the previous copyright update. 
Please change it to the one without specific year like in all other code files.

**Review by ishakhsuvarov — CHANGES_REQUESTED** [review]

**Comment by DanielRuf** [comment]
This is not very dynamic or flexible in my opinion.

I would either recommend the JS solution or the direct way with PHP. See http://updateyourfooter.com/

**Pull request merged**